website/content/docs/k8s/compatibility.mdx

```diff
@@ -29,6 +29,7 @@ apply to both Consul Enterprise and Consul community edition (CE).
 
 | Consul version | Compatible `consul-k8s` versions | Compatible Kubernetes versions | Compatible OpenShift versions |
 | -------------- | -------------------------------- | -------------------------------| ------------------------------|
+| 1.20.x         | 1.6.x                            | 1.28.x - 1.30.x                | 4.13.x - 4.16.x               |
```

Comment:
Before merge: verify that we've tested on Kubernetes 1.30.x and OpenShift 4.16.x

Reply:
Last I checked a few weeks ago, 4.16 may not be available on cloud providers yet - e.g. Azure still has it listed as coming soon: https://learn.microsoft.com/en-us/azure/openshift/support-lifecycle#azure-red-hat-openshift-release-calendar

We may have to stick with 4.15 until then

Reply:
RH docs state it’s possible at least on AWS: https://docs.openshift.com/container-platform/4.16/installing/installing_aws/ipi/installing-aws-default.html

Reply:
I've put this back to 4.15.x so this docs update can move ahead. This will allow us to do specific testing before we announce support for this brand new version of OpenShift that isn't yet supported by all clouds.